Feat: Tooltips

malaniz opened the pull request:
Feat: Tooltips for Withdrawal and Deposits

cazala reviewed (changes requested):
The tooltip should have a fixed position, otherwise it moves while scrolling and also appears already under the fold on certain screens (like my 13" screen):



![tooltip](https://user-images.githubusercontent.com/2781777/113614509-fc426500-9628-11eb-9205-fb1fcc867bf0.gif)

malaniz commented:
> The tooltip should have a fixed position, otherwise it moves while scrolling and also appears already under the fold on certain screens (like my 13" screen):
> 
> ![tooltip](https://user-images.githubusercontent.com/2781777/113614509-fc426500-9628-11eb-9205-fb1fcc867bf0.gif)

fixed! :D like `position: fixed`
